libs/features/pages/content-development/src/lib/features-pages-content-development.module.ts

```diff
@@ -4,10 +4,23 @@ import { CommonModule } from '@angular/common';
 import { ContentDevelopmentPageComponent } from './main/content-development-page/content-development-page.component';
 
 import { ContentDevelopmentPageRoutingModule } from './content-development.routing';
+import { ContentDevHeroSectionComponent } from './content-dev-hero-section/content-dev-hero-section.component';
+import { TextsModule } from '@elewa-website/elements/layout/texts';
+import { AppHeaderModule } from '@elewa-website/elements/layout/header';
+import { ButtonsModule } from '@elewa-website/elements/layout/buttons';
```

Comment:
Kindly, observe the order of imports; elements imports should appear before local imports

Reply:
@AnnaL001 *resolved